service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/HealthServiceImpl.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * JBoss, Home of Professional Open Source
+ * Copyright Red Hat, Inc., and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.aerogear.unifiedpush.service.impl;
+
+import org.jboss.aerogear.unifiedpush.jpa.dao.impl.JPAHealthDao;
+import org.jboss.aerogear.unifiedpush.service.HealthService;
+import org.jboss.aerogear.unifiedpush.service.impl.health.HealthDetails;
+import org.jboss.aerogear.unifiedpush.service.impl.health.Ping;
+import org.jboss.aerogear.unifiedpush.service.impl.health.PushNetwork;
+import org.jboss.aerogear.unifiedpush.service.impl.health.Status;
+
+import javax.ejb.AsyncResult;
+import javax.ejb.Asynchronous;
+import javax.ejb.Stateless;
+import javax.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Future;
+
+@Stateless
+public class HealthServiceImpl implements HealthService {
+    private static final PushNetwork[] PUSH_NETWORKS = new PushNetwork[] {
+            new PushNetwork("Google Cloud Messaging", "android.googleapis.com", 80),
+            new PushNetwork("Apple Push Network", "gateway.sandbox.push.apple.com", 2195),
+            new PushNetwork("Windows Push Network", "db3.notify.windows.com", 443)
+    };
```

Comment:
Well we support it for apns, the gateway/feedback host and ports can already be changed (by passing an ENV variable)